_inc/client/traffic/verification-services.jsx

```diff
@@ -123,20 +125,41 @@ export const VerificationServices = moduleSettingsForm(
 							{ __( 'Generate XML sitemaps' ) }
 						</ModuleToggle>
 						{
-							this.props.getOptionValue( 'sitemaps' ) && (
-								<FormFieldset>
-									<p>
-										<ExternalLink icon={ true } target="_blank" href={ sitemap_url }>{ sitemap_url }</ExternalLink>
-										<br />
-										<ExternalLink icon={ true } target="_blank" href={ news_sitemap_url }>{ news_sitemap_url }</ExternalLink>
+							this.props.isSiteVisibleToSearchEngines
+								? this.props.getOptionValue( 'sitemaps' ) && (
+									<FormFieldset>
+										<p>
+											<ExternalLink icon={ true } target="_blank" href={ sitemap_url }>{ sitemap_url }</ExternalLink>
+											<br />
+											<ExternalLink icon={ true } target="_blank" href={ news_sitemap_url }>{ news_sitemap_url }</ExternalLink>
+										</p>
+										<p className="jp-form-setting-explanation">{ __( 'Your sitemap is automatically sent to all major search engines for indexing.' ) }</p>
+									</FormFieldset>
+								)
+								: (
+									<p className="jp-form-setting-explanation">
+										{
+											__( 'Your site must be accessible by search engines for this feature to work properly. You can change this in {{a}}Reading Settings{{/a}}.', {
+												components: {
+													a: <a href={ this.props.siteAdminUrl + 'options-reading.php#blog_public' } />
```

Comment:
Let's get rid of the anchor-- it caused me to show up on the settings with my search engine settings hidden under the masterbar.